### PR TITLE
[zh] Sync KubeCon NA blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Translate KubeCon NA into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+对我们来说，一个值得骄傲的时刻是，我们的两位贡献者 Lin Sun 和 Faseela K
+荣获了由 CNCF 首席技术官 Chris Aniszczyk 在第二天的主题演讲中颁发的 CNCF 社区著名的
+["Chop Wood Carry Water" 奖](https://www.cncf.io/announcements/2023/11/08/cloud-native-computing-foundation-announces-2023-community-awards-winners/)。
+
+{{< image width="75%"
+    link="./chop-wood-carry-water.jpg"
+    caption="Chop Wood Carry Water 奖获得者 Faseela K 和 Lin Sun（左二、三）"
+    >}}
+
+我们的一些维护者和贡献者也进入了 CNCF 2023 年秋季大使名单，
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:106]:
```suggestion
我们的一些维护者和贡献者也入选了 CNCF 2023 年秋季大使名单，
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+例如 Lin Sun、Mitch Connors 和 Faseela K 等。
+
+{{< image width="75%"
+    link="./cncf-ambassadors.jpg"
+    caption="CNCF 大使合影。许多 Istio 维护者都在这张照片中！"
+    >}}
+
+[Istio KubeCon 维护者跟踪会议](https://sched.co/1R2tA) 由 TOC 成员 John Howard 和 Louis Ryan 主持，
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:114]:
```suggestion
指导委员会（TOC）成员 John Howard 和 Louis Ryan 主持的 [Istio KubeCon 维护者跟踪会议](https://sched.co/1R2tA)广受关注，
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+他们谈论了 Istio 当前正在进行的工作和未来路线图，引起了极大关注。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:115]:
```suggestion
会上他们谈论了 Istio 当前正在进行的工作和未来的技术路线图。
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+演讲中描述的技术以及由此产生的听众规模，强调了为什么 Istio 仍然是业界最受欢迎的服务网格。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:116]:
```suggestion
阐述了 Istio 所采用的技术、受众规模，强调了为什么 Istio 是业界持续最受欢迎的服务网格。
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+
+{{< image width="75%"
+    link="./maintainer-track.jpg"
+    alt="KubeCon 2023 北美站上的 Istio 维护者跟踪会议"
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:120]:
```suggestion
    alt="2023 北美 KubeCon 上的 Istio 维护者跟踪会议"
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+    >}}
+
+由 Lin Sun、Eric Van Norman、Steven Landow 和 Faseela K 主持的
+[Contribfest 实践开发与贡献研讨会](https://sched.co/1R2q7/)也广受好评。
+很高兴看到这么多人有兴趣为 Istio 做出贡献并在研讨会结束时提出他们的第一个 PR。
+
+{{< image width="75%"
+    link="./contrib-fest.jpg"
+    alt="KubeCon 2023 北美站中的 Istio Contribfest 研讨会"
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:129]:
```suggestion
    alt="2023 北美 KubeCon 上的 Contribfest Istio 研讨会"
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+    >}}
+
+由来自三个 CNCF 服务网格项目的维护者共同主持的备受期待的关于[服务网格之战的伤痕：技术、时机和权衡](https://sched.co/1R2ts)的小组讨论吸引了大批观众，
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:132]:
```suggestion
来自三个 CNCF 服务网格项目的维护者们牵头主持了备受期待的[服务网格之战的伤痕：技术、时机和权衡](https://sched.co/1R2ts)小组讨论，吸引了大批观众，
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+并产生了很多有趣的讨论。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:133]:
```suggestion
并进行了很多有趣的讨论。
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+
+{{< image width="75%"
+    link="./servicemesh-battle-scars-panel.jpg"
+    alt="KubeCon 2023 北美站中的服务网格之战的伤痕小组"
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:137]:
```suggestion
    alt="2023 北美 KubeCon 中的服务网格之战伤痕小组"
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 10]
```diff
@@ -0,0 +1,181 @@
...
+    >}}
+
+衷心感谢我们的白金赞助商 Google Cloud 对北美 Istio Day 的支持！
+最后且重要的一点是，我们要感谢 Istio Day 计划委员会成员的辛勤工作和支持！
+
+[2024 年 3 月巴黎见！](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/istio-day/)
+
+{{< image width="100%"
+    link="./istio-day-paris.jpg"
+    alt="Istio Day 2024 欧洲站"
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:180]:
```suggestion
    alt="2024 欧洲 Istio Day"
```
翻译通常是反序：
- en：活动 + 地点 + 时间
- zh：时间 + 地点 + 活动

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 11]
```diff
@@ -0,0 +1,181 @@
...
+
+在其他几场 KubeCon 演讲中，Istio 也成为了讨论的热门话题。
+以下是我们注意到的一些内容：
+
+* [走向边缘：使用 Istio 和 K8gb 创建全球分布式入口](https://sched.co/1R2o5/)
+* [幕后花絮：探索 Istio 的锁争用及其对 Expedia 计算平台的影响](https://sched.co/1R2uV)
+* [通过功能门控理清服务网格](https://sched.co/1R2v6)
+* [服务网格中的证书风格：原因和方法！](https://sched.co/1R2wC)
+
+Istio 在项目展馆设有一个展亭，被询问的大多数问题都与 Ambient 网格生产准备就绪的时间表有关。
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:148]:
```suggestion
Istio 在项目展馆设有一个展台，被询问的大多数问题都与 Ambient 网格生产准备就绪的时间表有关。
```
亭 是有顶的，istio 是个展台，下文同

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 12]
```diff
@@ -0,0 +1,181 @@
...
+
+{{< image width="75%"
+    link="./istio-booth-1.jpg"
+    caption="在 Istio 展亭进行讨论"
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:152]:
```suggestion
    caption="在 Istio 展台讨论"
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 13]
```diff
@@ -0,0 +1,181 @@
...
+    >}}
+
+我们很高兴看到在欧洲 Istio 展亭被提出的主要问题 —— CNCF 毕业时间表已经得到解答，
+并且我们向所有人保证，我们正在以同样的认真态度致力于 Ambient 网格。
+
+许多成员和维护者在我们的展亭提供了支持，帮助我们回答用户的所有问题。
+
+{{< image width="75%"
+    link="./istio-booth-2.jpg"
+    caption="Istio 展亭的成员和维护者"
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:162]:
```suggestion
    caption="Istio 展台的成员和维护者们"
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 14]
```diff
@@ -0,0 +1,181 @@
...
+    >}}
+
+我们展亭的另一个亮点是，我们有由 Microsoft、Solo.io、
+Stackgenie 和 Tetrate 赞助的新款 Istio T 恤供大家购买！
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:166]:
```suggestion
Istio 展台的另一个亮点是由 Microsoft、Solo.io、
Stackgenie 和 Tetrate 赞助的、供大家免费领取的新款 Istio T 恤！
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 15]
```diff
@@ -0,0 +1,181 @@
...
+
+{{< image width="75%"
+    link="./istio-t-shirts.jpg"
+    caption="新款 Istio T 恤"
+    >}}
+
+衷心感谢我们的白金赞助商 Google Cloud 对北美 Istio Day 的支持！
+最后且重要的一点是，我们要感谢 Istio Day 计划委员会成员的辛勤工作和支持！
```
Comment (windsonsea) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:174]:
```suggestion
最后且重要的一点是，我们要感谢 Istio Day 技术程序委员会成员，感谢他们的辛勤工作和支持！
```
跟开头的译法要一致

**Review by windsonsea — COMMENTED** [review]
some text suggestions

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 16]
```diff
@@ -0,0 +1,181 @@
...
+若您想重温展会风采，请查阅 4 月份举办的[欧洲 Istio Day](/zh/blog/2023/istio-at-kubecon-eu/)、
+[2023 虚拟 IstioCon](https://events.istio.io/) 活动以及于
+9 月 26 日在中国上海举办的[2023 中国 IstioCon](/zh/blog/2023/istiocon-china/)。
+
+{{< image width="75%"
+    link="./welcome.jpg"
+    alt="2023 北美 Istio Day 欢迎标牌"
+    >}}
+
+Istio Day 活动以 Program Committee（技术程序委员会）主席 Faseela K 和 Zack Butcher 的开幕致辞拉开帷幕。
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:24]: 这个 Program Committee 的翻译感觉不太对？没有技术两个字吧

Comment (hanxiaop) [reply content/zh/blog/2023/istio-at-kubecon-na/index.md:24]:
```suggestion
Istio Day 活动以 Program Committee（程序委员会）主席 Faseela K 和 Zack Butcher 的开幕致辞拉开帷幕。
```

**Comment by windsonsea** [comment]
可以借鉴参考[云原生社区的推文](https://mp.weixin.qq.com/s/nvKQ0qjlZAgkuFNdzDKPkA)，他们动作真快，已经翻译好发表了

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 17]
```diff
@@ -0,0 +1,181 @@
...
+{{< image width="75%"
+    link="./welcome.jpg"
+    alt="2023 北美 Istio Day 欢迎标牌"
+    >}}
+
+Istio Day 活动以 Program Committee（程序委员会）主席 Faseela K 和 Zack Butcher 的开幕致辞拉开帷幕。
+两位主席首先肯定并认可了所有社区贡献者、维护者、发布经理以及用户们所完成的日常工作，
+并为 Istio 最杰出的贡献者和社区大咖们颁发了一些奖项。
+Rob Salmond 和 Andrea Ma 因其在 Istio 社区做出的无私贡献而得享赞誉，
+还在大屏上展示了过去 6 个月内排名前 20 的贡献者名单。
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:28]:
```suggestion
过去 6 个月中的前 20 名贡献者也被点名表扬。
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 18]
```diff
@@ -0,0 +1,181 @@
...
+Istio 展台的另一个亮点是由 Microsoft、Solo.io、
+Stackgenie 和 Tetrate 赞助的、供大家免费领取的新款 Istio T 恤！
+
+{{< image width="75%"
+    link="./istio-t-shirts.jpg"
+    caption="新款 Istio T 恤"
+    >}}
+
+衷心感谢我们的白金赞助商 Google Cloud 对北美 Istio Day 的支持！
+最后且重要的一点是，我们要感谢 Istio Day 技术程序委员会成员，感谢他们的辛勤工作和支持！
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:174]: 这个原文也是 Program Committee

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 19]
```diff
@@ -0,0 +1,181 @@
...
+其中 Karim Lakhani 解释了 Intuit 的现代 SaaS 平台部署的多个包括 Istio 在内的云原生项目。
+当 Mitch Connors 和 Christian Hernandez
+在实时公共站点上使用可公开访问的可用性监控进行[使用 Argo 升级 Istio Ambient 网格的现场演示](https://www.youtube.com/watch?v=o71PJAqy4P8)时，观众们非常兴奋。
+
+{{< image width="75%"
+    link="./istioday-session-1.jpg"
+    caption="Istio Day 的丰富会议内容"
+    >}}
+
+在随后与来自 Microsoft 的 Jackie Elliot 的更加关注安全性的会谈活动中，
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:67]:
```suggestion
此次活动在随后的演讲中更加关注安全性，首先是来自 Microsoft 的 Jackie Elliot
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 20]
```diff
@@ -0,0 +1,181 @@
...
+深入研究了 [Istio 身份](https://www.youtube.com/watch?v=QjmUDNXyckQ)，
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:68]:
```suggestion
深入研究了 [Istio 身份认证](https://www.youtube.com/watch?v=QjmUDNXyckQ)，
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 21]
```diff
@@ -0,0 +1,181 @@
...
+随后在来自 Speedscale 的 Kush Mansing 进行的闪电演讲中，
+展示了在 Istio 中[使用肆意代码运行服务带来的影响](https://www.youtube.com/watch?v=G6Y9JLnej0o)。
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:70]:
```suggestion
展示了在 Istio 中[使用任意代码运行服务带来的影响](https://www.youtube.com/watch?v=G6Y9JLnej0o)。
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 22]
```diff
@@ -0,0 +1,181 @@
...
+我们接着进行了[来自 Xiangfeng Zhu 的闪电演讲](https://www.youtube.com/watch?v=lHUXvtSWdtQ)，
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:71]:
```suggestion
我们还有[来自 Xiangfeng Zhu 的闪电演讲](https://www.youtube.com/watch?v=lHUXvtSWdtQ)，
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 23]
```diff
@@ -0,0 +1,181 @@
...
+他是来自华盛顿大学的博士生，在演讲中展示了一种用于分析和预测 Istio 性能开销的工具 。
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:72]:
```suggestion
他在其中展示了一个用于分析和预测Istio的性能开销的工具。
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 24]
```diff
@@ -0,0 +1,181 @@
...
+
+此次活动在随后的演讲中更加关注安全性，首先是来自 Microsoft 的 Jackie Elliot
+深入研究了 [Istio 身份认证](https://www.youtube.com/watch?v=QjmUDNXyckQ)，
+随后在来自 Speedscale 的 Kush Mansing 进行的闪电演讲中，
+展示了在 Istio 中[使用任意代码运行服务带来的影响](https://www.youtube.com/watch?v=G6Y9JLnej0o)。
+我们还有来自 University of Washington 的博士生 Xiangfeng Zhu 的[闪电演讲](https://www.youtube.com/watch?v=lHUXvtSWdtQ)，
+他是来自华盛顿大学的博士生，在演讲中展示了一种用于分析和预测 Istio 性能开销的工具 。
+
+来自 Kiali 的维护者 Jay Shaughnessy 和 Nick Fox 的[演讲](https://www.youtube.com/watch?v=MX-Sym2EkGI)非常有趣，
+其中演示了许多使用 Kiali 高级方法更好地对 Istio 进行调试的用例。
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:75]:
```suggestion
因为它展示了许多使用 Kiali 进行更好的 Istio 用例调试的高级方法。
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md (hunk 18 above)
Comment (wilsonwu) [reply content/zh/blog/2023/istio-at-kubecon-na/index.md]: Fixed.

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 25]
```diff
@@ -0,0 +1,181 @@
...
+来自 Zeta 的 Ekansh Gupta 和来自 Reskill 的 Nirupama Singh
+发表了另一场[最终用户演讲，解释了在生产部署中升级 Istio 时的最佳实践](https://www.youtube.com/watch?v=dl0sESwwm9c)。
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:77]:
```suggestion
发表了另一场[最终用户演讲，解释了在生产部署中升级 Istio 的最佳实践](https://www.youtube.com/watch?v=dl0sESwwm9c)。
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md (hunk 23 above)
Comment (wilsonwu) [reply content/zh/blog/2023/istio-at-kubecon-na/index.md]: Fixed.

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 26]
```diff
@@ -0,0 +1,181 @@
...
+由 Lin Sun、Eric Van Norman、Steven Landow 和 Faseela K 主持的
+[Contribfest 实践开发与贡献研讨会](https://sched.co/1R2q7/)也广受好评。
+很高兴看到这么多人有兴趣为 Istio 做出贡献并在研讨会结束时提出他们的第一个 PR。
+
+{{< image width="75%"
+    link="./contrib-fest.jpg"
+    alt="2023 北美 KubeCon 上的 Contribfest Istio 研讨会"
+    >}}
+
+来自三个 CNCF 服务网格项目的维护者们牵头主持了备受期待的[服务网格之战的伤痕：技术、时机和权衡](https://sched.co/1R2ts)小组讨论，吸引了大批观众，
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:132]:
```suggestion
来自三个 CNCF 服务网格项目的维护者们牵头主持了备受期待的[服务网格战痕：技术、时机和权衡](https://sched.co/1R2ts)小组讨论，吸引了大批观众，
```

File: content/zh/blog/2023/istio-at-kubecon-na/index.md [hunk 27]
```diff
@@ -0,0 +1,181 @@
...
+
+在其他几场 KubeCon 演讲中，Istio 也成为了讨论的热门话题。
+以下是我们注意到的一些内容：
+
+* [走向边缘：使用 Istio 和 K8gb 创建全球分布式入口](https://sched.co/1R2o5/)
+* [幕后花絮：探索 Istio 的锁争用及其对 Expedia 计算平台的影响](https://sched.co/1R2uV)
+* [通过功能门控理清服务网格](https://sched.co/1R2v6)
+* [服务网格中的证书风格：原因和方法！](https://sched.co/1R2wC)
+
+Istio 在项目展馆设有一个展台，被询问的大多数问题都与 Ambient 网格生产准备就绪的时间表有关。
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-at-kubecon-na/index.md:148]:
```suggestion
Istio 在项目展馆设有一个展台，大多数问题都围绕着 Ambient 何时准备好投入生产。
```

**Review by hanxiaop — COMMENTED** [review]
Looks good!

**Review by hanxiaop — APPROVED** [review]

**Review by windsonsea — APPROVED** [review]
Thanks

**Pull request merged**